tests/cephadm/test_cephadm_shell.py

```diff
@@ -0,0 +1,58 @@
+from ceph.ceph_admin import CephAdmin
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def create_mon_daemon_file(node):
+    """
+    Method create mon dir in /var/lib/ceph and create a daemon specific folder/file, like ceph-doc or ceph-test
+    Args:
+        node: node object
+    Returns:
+        boolean
+    """
+
+    try:
+        out, _ = node.exec_command(
+            cmd="mkdir -p /var/lib/ceph && touch /var/lib/ceph/ceph-test", sudo=True
+        )
+        return 0
+    except Exception as e:
+        log.error(e)
+        return 1
+
+
+def run(ceph_cluster, **kw):
+    """Verification cephadm shell when running from a non-monitor node
```

Comment:
HI Mohit, how are we selecting non-monitor node?

Reply:
Thanks Manohar

Below set of method select "installer node"(non-monitor node).

L45 and L47

instance = CephAdmin(cluster=ceph_cluster, **config)
log.info("Creating a mon dir and daemon specfic file")
create_mon_daemon_file(instance.installer)

Reply:
Thanks.